packages/edit-site/src/components/header/tools-more-menu-group/index.js

```diff
@@ -6,20 +6,15 @@ import { isEmpty } from 'lodash';
 /**
  * WordPress dependencies
  */
-import { createSlotFill, MenuGroup } from '@wordpress/components';
-import { __ } from '@wordpress/i18n';
+import { createSlotFill } from '@wordpress/components';
 
 const { Fill: ToolsMoreMenuGroup, Slot } = createSlotFill(
 	'EditSiteToolsMoreMenuGroup'
 );
 
 ToolsMoreMenuGroup.Slot = ( { fillProps } ) => (
 	<Slot fillProps={ fillProps }>
-		{ ( fills ) =>
-			! isEmpty( fills ) && (
-				<MenuGroup label={ __( 'Tools' ) }>{ fills }</MenuGroup>
-			)
-		}
+		{ ( fills ) => ! isEmpty( fills ) && <>{ fills }</> }
```

Comment:
Is the fragment wrapper required?

Reply:
You're right. There's no need for the fragment. Removed in c761f03